data-es2016plus.js

```diff
@@ -3013,15 +3012,12 @@ exports.tests = [
           firefox2: false,
           firefox57: false,
           firefox58: true,
-          opera10_50: false,
           chrome65: chrome.harmony,
           chrome66: true,
           safari11_1: true,
           safaritp: true,
           webkit: true,
           duktape2_2: false,
```

Comment:
This should be `true` according to @jridgewell's [comment](https://github.com/kangax/compat-table/pull/1610#issuecomment-619465950)

Reply:
Sorry, I misspoke. I confirmed that this failed, as the original value did.